### PR TITLE
Try to more explicit state SEPA only for bank transfers

### PR DESCRIPTION
**Pull request by frlan**

### CODE REVIEW

**Comment by eht16** [comment]
Was there any confusion?

Now, I'm confused: should this mean bank transfers are only possible in Europe at all or should it mean bank transfers using SEPA are only supported in Europe?

**Comment by frlan** [comment]
Well... it seems to some donators it was not clear enough that bank transfer require SEPA-support, which is not given in e.g. North American banks. I really hoped that this makes it more clear.

File: page_content/about/donate.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ More than financial support, the Geany project needs help with:
 If you want to donate money to the Geany project, it will be used to support further development of Geany, to pay running costs for hosting and domains and support presenting Geany on various [Open Source Conferences][5].
 
 There are different options to donate, including bank transfer and Liberapay.
-Bank transfer to the [Geany e.V.][8] account is the preferred way for donating money as the money is transferred directly without a third party involved and additional fees can be avoided.
+Bank transfer to the [Geany e.V.][8] account (only SEPA / Europe) is the preferred way for donating money as the money is transferred directly without a third party involved and additional fees can be avoided.
```
Comment (eht16) [inline page_content/about/donate.md:19]:
```suggestion
Bank transfer to the [Geany e.V.][8] account is the preferred way for donating money as the money is transferred directly without a third party involved and additional fees can be avoided.
Please note bank transfers are only possible if your bank participates in [SEPA][10], usually this applies to banks in most European countries.
```

To me, this is clearer. What do you think?
`[SEPA][10]` should be a reference to https://en.wikipedia.org/wiki/Single_Euro_Payments_Area and need to be added at the bottom.

Comment (frlan) [reply page_content/about/donate.md:19]: Yes, should make the trick.

Comment (eht16) [reply page_content/about/donate.md:19]: Do you mind changing it? :D

Comment (frlan) [reply page_content/about/donate.md:19]: Just saw the comment -- github is just way to noisy by default. Will update it next chance.

**Comment by eht16** [comment]
Looks good, fine to squash+merge for me.

**Pull request merged**